python/tvm/relay/op/memory/memory.py

```diff
@@ -104,6 +104,21 @@ def shape_func(func, inputs, outputs, dependent=False):
     """
     return _make.shape_func(func, inputs, outputs, dependent)
 
+def shape_of(expr):
```

Comment:
since we already call it `vm.shape_of` in the cpp, I think we should also put `shape_of` into the folder of op/vm.

Reply:
Yeah, I agree with you. The reason why I left it here was because I think we may need to move all these ops under op/vm. How do you think?

Reply:
I think memory related ops should be still in the `memory` namespace such that potentially other runtime can reuse it.

Reply:
yeah, maybe `invoke_tvm_op` should be ported there as well later?